aws-rds-dbclusterendpoint/src/main/java/software/amazon/rds/dbclusterendpoint/UpdateHandler.java

```diff
@@ -0,0 +1,52 @@
+package software.amazon.rds.dbclusterendpoint;
+
+import software.amazon.awssdk.services.rds.RdsClient;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ProxyClient;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+import software.amazon.rds.common.handler.HandlerConfig;
+import software.amazon.rds.common.handler.Tagging;
+
+
+public class UpdateHandler extends BaseHandlerStd {
+
+
+    public UpdateHandler() {
+        this(HandlerConfig.builder()
+                .backoff(BACKOFF_DELAY)
+                .build());
+    }
+
+    public UpdateHandler(HandlerConfig config) {
+        super(config);
+    }
+    protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
+            final AmazonWebServicesClientProxy proxy,
+            final ResourceHandlerRequest<ResourceModel> request,
+            final CallbackContext callbackContext,
+            final ProxyClient<RdsClient> proxyClient,
+            final Logger logger
+    ) {
+        final ResourceModel desiredModel = request.getDesiredResourceState();
+
+        final Tagging.TagSet previousTags = Tagging.TagSet.builder()
+                .systemTags(Tagging.translateTagsToSdk(request.getPreviousSystemTags()))
+                .stackTags(Tagging.translateTagsToSdk(request.getPreviousResourceTags()))
+                .resourceTags(Translator.translateTagsToSdk(request.getPreviousResourceState().getTags()))
+                .build();
+
+        final Tagging.TagSet desiredTags = Tagging.TagSet.builder()
+                .systemTags(Tagging.translateTagsToSdk(request.getSystemTags()))
+                .stackTags(Tagging.translateTagsToSdk(request.getDesiredResourceTags()))
+                .resourceTags(Translator.translateTagsToSdk(request.getDesiredResourceState().getTags()))
+                .build();
+
+        return ProgressEvent.progress(desiredModel, callbackContext)
+                // Since modifying any property if DBClusterEndpoint causes interruption,
+                // we should only update tags. Updating any other properties should require replacement
```

Comment:
Could you please double-check this is the case? I see modify request accepting these parameters.